src/Prefixer.js

```diff
@@ -82,12 +98,42 @@ export function prefix (value, length) {
 						return ~indexof(value, 'stretch') ? prefix(replace(value, 'stretch', 'fill-available'), length) + value : value
 				}
 			break
+		// grid-(column|row)
```

Comment:
we shouldn't use inline arrow function closure, also possibly more size optimisations, i.e hoisting the regex. I'll submit these and more to this branch this weekend.